### DISCUSSION TITLE
FR : CLI to open a directory

### DISCUSSION THREAD

**Discussion by arnaud-feldmann** [ddterm/gnome-shell-extension-ddterm#135] [Ideas]
Hi and thanks for your work ! You've got the best extension,

I have seen the different methods offered with the DBus, and know that I can toggle with
dbus-send --session --type=method_call  --print-reply --dest='org.gnome.Shell' /org/gnome/Shell/Extensions/ddterm com.github.amezin.ddterm.Extension.Toggle

However, I'd like to use this dropdown terminal with, by example, the nautilus "open in a terminal". That would need some way to open a directory through the CLI.

Is there any way to do so for now ?

Otherwise, that would be a very nice feature request !

**Comment by amezin** [comment]
This will need to be added in multiple areas:

- a new method in the extension's D-Bus interface
- GAction in AppWindow
- that GAction should be re-exported by Application and, in turn, called by the extension.

Should be doable. Similar to Activate/Toggle, but with a parameter now.
